client/webserver/webserver.go

```diff
@@ -625,6 +627,37 @@ func New(cfg *Config) (*WebServer, error) {
 	return s, nil
 }
 
+// fetchLatestVersion is a helper function to retrieve the latest version of the app
+// from github.
+func (w *WebServer) fetchLatestVersion(ctx context.Context) {
+	if w.latestVersion != "" {
+		return // already set
+	}
+
+	ticker := time.NewTicker(30 * time.Second)
```

Comment:
Idk if we need to check so often, maybe once or twice a day?

Reply:
Currently will hit rate limits, as I was able to https://docs.github.com/en/rest/using-the-rest-api/rate-limits-for-the-rest-api?apiVersion=2022-11-28#primary-rate-limit-for-unauthenticated-users